src/systemctl/systemctl.c

```diff
@@ -8079,6 +8085,7 @@ static int systemctl_parse_argv(int argc, char *argv[]) {
                 { "message",             required_argument, NULL, ARG_MESSAGE             },
                 { "show-transaction",    no_argument,       NULL, 'T'                     },
                 { "what",                required_argument, NULL, ARG_WHAT                },
+                { "reboot-arg",          required_argument, NULL, ARG_REBOOT_ARG          },
```

Comment:
in unit files we generally spell the corresponding option out, i.e. RebootArgument= rather than just RebootArg=. I think we should do this here too. This is a long argument, and not used all the time, hence saving the "ument" is not worth it.